hardhat.config.js

```diff
@@ -15,10 +15,13 @@ const accounts = {
     accountsBalance: "100000000000000000000000000",
 };
 
-const ALCHEMY_API_KEY = process.env.ALCHEMY_API_KEY;
-let GOERLI_MNEMONIC = process.env.GOERLI_MNEMONIC;
-if (!GOERLI_MNEMONIC) {
-    GOERLI_MNEMONIC = accounts.mnemonic;
+const ALCHEMY_API_KEY_MAINNET = process.env.ALCHEMY_API_KEY_MAINNET;
+const ALCHEMY_API_KEY_MATIC = process.env.ALCHEMY_API_KEY_MATIC;
+const ALCHEMY_API_KEY_GOERLI = process.env.ALCHEMY_API_KEY_GOERLI;
+const ALCHEMY_API_KEY_MUMBAI = process.env.ALCHEMY_API_KEY_MUMBAI;
```

Comment:
This will be ultimately adapted in the form of distinguishing API keys on each network, as their number will grow.